Sources/SentryCrash/Recording/Tools/SentryCrashStackCursor.c

```diff
@@ -50,6 +50,7 @@ sentrycrashsc_resetCursor(SentryCrashStackCursor *cursor)
     cursor->stackEntry.imageName = NULL;
     cursor->stackEntry.symbolAddress = 0;
     cursor->stackEntry.symbolName = NULL;
+    cursor->useDlAddr = false;
```

Comment:
This is what worries me the most.
Im not sure whether `sentrycrashsc_resetCursor` is called in every scenario.
If `uselDlAddr` is no initialized to false during crashes, the value will be random and 'dladdr` may be used for crashes which is not desired.

Reply:
I opened a PR to refactor this and remove this extra property https://github.com/getsentry/sentry-cocoa/pull/2659.

Reply:
Perfect! This solution is definitely safer.